kolibri/plugins/perseus_viewer/assets/src/translator.js

```diff
@@ -0,0 +1,165 @@
+import { createTranslator } from 'kolibri.utils.i18n';
+
+const translator = createTranslator('PerseusInternalMessages', {
+  Hints: 'Hints',
+  'Get another hint': 'Get another hint',
+  'Hint #{ pos }': 'Hint #{ pos }',
+  'an integer, like $6$': 'an integer, like $6$',
+  'a *proper* fraction, like $1/2$ or $6/10$': 'a *proper* fraction, like $1/2$ or $6/10$',
+  'a *simplified proper* fraction, like $3/5$': 'a *simplified proper* fraction, like $3/5$',
+  'an *improper* fraction, like $10/7$ or $14/8$': 'an *improper* fraction, like $10/7$ or $14/8$',
+  'a *simplified improper* fraction, like $7/4$': 'a *simplified improper* fraction, like $7/4$',
+  'a mixed number, like $1 3/4$': 'a mixed number, like $1\\\\ 3/4$',
+  'an *exact* decimal, like $0.75$': 'an *exact* decimal, like $0.75$',
+  'a percent, like $12.34%$': 'a percent, like $12.34\\\\%$',
+  'a multiple of pi, like $12 text{pi}$ or $2/3 text{pi}$':
+    'a multiple of pi, like $12\\\\ \\\\text{pi}$ or $2/3\\\\ \\\\text{pi}$',
+  '**Your answer should be** ': '**Your answer should be** ',
+  'Your answer:': 'Your answer:',
+  '0 solutions': '0 solutions',
+  'Finite solutions': 'Finite solutions',
```

Comment:
When running the build script, my version of this file got prettified which introduced an undesirable diff:


```diff
-  Hints: 'Hints',
-  'Get another hint': 'Get another hint',
-  'Hint #{ pos }': 'Hint #{ pos }',
-  'an integer, like $6$': 'an integer, like $6$',
-  'a *proper* fraction, like $1/2$ or $6/10$': 'a *proper* fraction, like $1/2$ or $6/10$',
-  'a *simplified proper* fraction, like $3/5$': 'a *simplified proper* fraction, like $3/5$',
-  'an *improper* fraction, like $10/7$ or $14/8$': 'an *improper* fraction, like $10/7$ or $14/8$',
-  'a *simplified improper* fraction, like $7/4$': 'a *simplified improper* fraction, like $7/4$',
-  'a mixed number, like $1 3/4$': 'a mixed number, like $1\\\\ 3/4$',
-  'an *exact* decimal, like $0.75$': 'an *exact* decimal, like $0.75$',
-  'a percent, like $12.34%$': 'a percent, like $12.34\\\\%$',
-  'a multiple of pi, like $12 text{pi}$ or $2/3 text{pi}$':
-    'a multiple of pi, like $12\\\\ \\\\text{pi}$ or $2/3\\\\ \\\\text{pi}$',
-  '**Your answer should be** ': '**Your answer should be** ',
-  'Your answer:': 'Your answer:',
-  '0 solutions': '0 solutions',
-  'Finite solutions': 'Finite solutions',
+  "Hints": "Hints",
+  "Get another hint": "Get another hint",
+  "Hint #{ pos }": "Hint #{ pos }",
+  "an integer, like $6$": "an integer, like $6$",
+  "a *proper* fraction, like $1/2$ or $6/10$": "a *proper* fraction, like $1/2$ or $6/10$",
+  "a *simplified proper* fraction, like $3/5$": "a *simplified proper* fraction, like $3/5$",
+  "an *improper* fraction, like $10/7$ or $14/8$": "an *improper* fraction, like $10/7$ or $14/8$",
+  "a *simplified improper* fraction, like $7/4$": "a *simplified improper* fraction, like $7/4$",
+  "a mixed number, like $1 3/4$": "a mixed number, like $1\\\\ 3/4$",
+  "an *exact* decimal, like $0.75$": "an *exact* decimal, like $0.75$",
+  "a percent, like $12.34%$": "a percent, like $12.34\\\\%$",
+  "a multiple of pi, like $12 text{pi}$ or $2/3 text{pi}$": "a multiple of pi, like $12\\\\ \\\\text{pi}$ or $2/3\\\\ \\\\text{pi}$",
+  "**Your answer should be** ": "**Your answer should be** ",
+  "Your answer:": "Your answer:",
+  "0 solutions": "0 solutions",
+  "Finite solutions": "Finite solutions",
```

Reply:
I can lint the file before writing it using kolibri-tools, will update.